lib/streamlit/config.py

```diff
@@ -794,6 +794,22 @@ def _server_address() -> Optional[str]:
     type_=bool,
 )
 
+_create_option(
+    "server.ipHeader",
+    description="""When streamlit is deployed behind a reverse proxy, you can get user ip by request
+                header, for example 'X-Real-IP' or 'X-Forwarded-For'.""",
```

Comment:
nit: could we be a bit more consistent with the `description` docstring formatting of other config options? Same comment for the config option below